docs/latest/modules/en/pages/setup/otel/otelmappings/README.adoc

```diff
@@ -2,5 +2,32 @@ ifdef::ss-ff-stackpacks2_enabled[]
 = Getting Started with Open Telemetry Mappings
 :revdate: 2025-11-26
 :page-revdate: {revdate}
-:description: Overview of Open Telemetry  Mappings
+:description: Overview of Open Telemetry Mappings
+
+SUSE Observability can automatically derive application and infrastructure topology from OpenTelemetry (OTel) telemetry data.
+This is achieved through OpenTelemetry topology mappings, which declaratively transform trace and metric signals into topology components (vertices) and relations (edges).
+
+These mappings allow you to describe what topology should be created from telemetry.
+They are evaluated as part of the topology pipeline inside the OpenTelemetry Collector.
+The resulting component and relation objects are emitted downstream and later materialised by the SUSE Observability platform once the required vertices are available.
+
+== Overview
+
+At a high level, topology generation from OpenTelemetry works as follows:
+
+* OpenTelemetry telemetry (metrics and/or traces) is received by the Collector
+* The topology pipeline evaluates configured mappings against the telemetry
```

Comment:
The collector evaluates I would say. The whole thing is the pipeline so that's a bit unclear.